pynest/nest/hl_api.py

```diff
@@ -61,11 +57,17 @@ def _import_libs(mod_file, mod_dict, path, prefix, ignore_modules=frozenset()):
             continue  # not a regular python module
 
         pkg_name = "{}.{}".format(prefix, name[:-3])
-        if pkg_name in ignore_modules:
+        if pkg_name in ignore:
             continue  # this package is not to be imported dynamically
 
-        pkg = __import__(pkg_name, mod_dict, locals(), ['*'], 1)
-        _import_names(pkg, mod_dict)
+        # construct from .pkg_name import * ; this is the old 'exec(string)'
```

Comment:
Thanks for the hint. I think you can remove the `; this is the old 'exec(string)'` part from this comment now.